tests/validator/validator.cpp

```diff
@@ -1981,26 +1981,60 @@ TEST(Validator, unitUserCreatedUnitsBananasAndApples)
     EXPECT_EQ_ISSUES(expectedIssues, validator);
 }
 
-TEST(Validator, unitIllDefinedEquivalentUnits)
+TEST(Validator, unitMissingEquivalentUnits)
 {
-    const std::vector<std::string> expectedIssues = {
+    const std::vector<std::string> expectedIssues1 = {
```

Comment:
You have split the test, so no need to rename `expectedIssues` to `expectedIssues1` anymore.

Reply:
541b8354